
# Here's a description of our workðŸš€ 

### You can visit the team's sheet to get more insider details - [Team-Ramachandran](https://bit.ly/2YFH5ji)

- Initially, each team identified and characterized (physiochemical) a minimum of 10 proteins of COVID19 or related SARS and MERS virus proteins from databases liked PubMed Protein and UniProt. We then calculated the properties (protein-parameters) of those uncharacterized proteins like Molecular Weight,Hydrophilicty, Hydrophobicity, Isoelectric PH and Number of R, H, K, C (in %) residues. We used a variety of tools, likes of which incluse ProtPARAM [See team-sheet for more].

- On the side we were tasked to find the HLA binding predictions in Asian and African populations for the virus and ran BLAST analysis to find coverage between our protein FASTA files from UniProt and tempLate FASTA files from the homology modelling servers.

- We then proceeded on Homology Modelling of those 10 proteins with 10 different homology modelling servers and then proceeded to make the structure models. We calculated their stabilities and ramachandran analysis, refined the models and ran checks and finally went our way to pick the most stable ones (which have high coverage and minimized energy). Currently, we are working on 4 out of those proteins. The 4 proteins are selected based on different stoichiometric, geometric, and thermodynamic parameters and selecting the best protein model. 

-Having selected the best 4 from each team, we are working on optimizing the 4 and also doing HLA-binding predictions to identify possible epitopes in the protein Sequences. Alongside, we are carrying out HLA-protein docking, using the HLA-1 alleles we identified earlier. We started docking with MHC to protein and surface epitopes.

-We worked on depositing these proteins (most stable 4) on different databases such as proteopedia, model archive, wwPDB, etc.

